src/main/java/com/gtnewhorizons/gtnhgradle/PropertiesConfiguration.java

```diff
@@ -341,6 +345,47 @@ Enables using modern Java syntax (up to version 17) via Jabel, while still targe
             """)
     public int forceToolchainVersion = -1;
 
+    /** See annotation */
+    @Prop(
+        name = "downgradeTargetVersion",
+        isSettings = false,
+        preferPopulated = false,
+        required = false,
+        docComment = """
+            Target JVM version for JVM Downgrader bytecode downgrading.
+            Valid values: 8, 11, 17. Only used when enableModernJavaSyntax = jvmDowngrader
```

Comment:
Any reason to restrict this to 8,11,17? While other versions might not be recommended, I don't think there's any need for this to be a hard constraint